tests/integ/modin/pivot/test_pivot_single.py

```diff
@@ -240,9 +243,186 @@ def test_pivot_on_inline_data_using_temp_table():
+                columns = [None if not named_columns else "columns"] + columns
+            else:
+                levels = codes = [[]]
+            native_df = native_pd.DataFrame(
+                index=pd.Index([], name=index if isinstance(index, str) else index[0]),
+                columns=pd.MultiIndex(levels=levels, codes=codes, names=columns),
+            )
+        else:
+            native_df = native_df.pivot_table(
+                index=index, columns=columns, margins=margins
+            )
+        assert_snowpark_pandas_equal_to_pandas(
+            snow_df, native_df, check_index_type=False, check_column_type=False
+        )
+
+
+@pytest.mark.parametrize("margins", [True, False])
+@sql_count_checker(query_count=1)
+@pytest.mark.parametrize("named_columns", [True, False])
+def test_pivot_empty_frame_no_values(margins, named_columns):
+    columns = list("abcd")
+    if named_columns:
+        columns = pd.Index(columns, name="columns")
+    snow_df, native_df = create_test_dfs(columns=columns)
+    snow_df = snow_df.pivot_table(index=["c", "d"], columns=["a", "b"], margins=margins)
+    if margins:
+        # When margins is True, and there are no values (i.e. all of the
+        # columns in the DataFrame are passed in to either the `index`
+        # or `columns` parameter), pandas errors out. We return an empty
+        # DataFrame instead.
+        with pytest.raises(TypeError, match="'str' object is not callable"):
+            native_df.pivot_table(index=["c", "d"], columns=["a", "b"], margins=margins)
+        levels = codes = [[], []]
+        ind = pd.MultiIndex(levels=levels, codes=codes, names=["c", "d"])
+        levels = codes = [[], [], []]
+        cols = pd.MultiIndex(
+            levels=levels,
+            codes=codes,
+            names=[None if not named_columns else "columns", "a", "b"],
+        )
+        native_df = native_pd.DataFrame(index=ind, columns=cols)
+    else:
+        native_df = native_df.pivot_table(
+            index=["c", "d"], columns=["a", "b"], margins=margins
+        )
+    assert_snowpark_pandas_equal_to_pandas(
+        snow_df, native_df, check_index_type=False, check_column_type=False
+    )
+
+
+@pytest.mark.skip(reason="No support for multi-indexes in pivot yet.")
+@pytest.mark.parametrize(
+    "index, columns",
+    [
+        (None, ("a", 1)),
+        (None, [("a", 1)]),
+        (None, [("a", 1), ("b", 2)]),
+        (None, [("a", 1), ("b", 2), ("c", 3)]),
+        (("d", 4), ("a", 1)),
+        (("d", 4), [("a", 1)]),
+        (("d", 4), [("a", 1), ("b", 2)]),
+        (("d", 4), [("a", 1), ("b", 2), ("c", 3)]),
+        ([("d", 4)], ("a", 1)),
+        ([("d", 4)], [("a", 1)]),
+        ([("d", 4)], [("a", 1), ("b", 2)]),
+        ([("d", 4)], [("a", 1), ("b", 2), ("c", 3)]),
+    ],
+)
+@pytest.mark.parametrize("margins", [True, False])
+@pytest.mark.parametrize("named_columns", [True, False])
+def test_pivot_empty_frame_snow_1013918_multiindex(
```

Comment:
it is little bit wired that we are adding test that are skipped, was it because mutliIndex is not supported by snowpark pandas yet, and what you are adding there should also works for multiIndex? Can you create a ticket for the case that currently is not supported yet, and add the ticket number in the reason

Reply:
Sure - the test is skipped because multiindex isn't supported yet, but I want to make sure we catch this case when we do implement multiindex. Will look for (or create) a ticket for this, and include in the description!